test/services/utils/IssueUtil.test.ts

```diff
@@ -0,0 +1,208 @@
+import {
+  findSigLabel,
+  isChallengeIssue,
+  needHelp,
+  checkIsInAssignFlow,
+  isClosed,
+  findMentorAndScore,
+} from "../../../src/services/utils/IssueUtil";
+import {
+  CHALLENGE_PROGRAM_LABEL,
+  HELP_WANTED_LABEL,
+} from "../../../src/commands/labels";
+
+describe("Issue Util", () => {
+  test("issue have sig label", () => {
```

Comment:
It think the description should be `test("sig label can be found")`.

It is best to check if your other test descriptions are appropriate.

Reply:
ok

Reply:
It's best to use `can` or `could` in all descriptions.